app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java

```diff
@@ -74,6 +77,33 @@ public boolean onOptionsItemSelected(MenuItem item) {
             case android.R.id.home:
                 finish();
                 return true;
+            case R.id.share_speakers_url:
+                Intent sendIntent = new Intent();
+                sendIntent.setAction(Intent.ACTION_SEND);
+                sendIntent.putExtra(Intent.EXTRA_SUBJECT, getResources().getString(R.string.subject));
+                StringBuilder message = new StringBuilder();
+                message.append(selectedSpeaker.getName())
+                        .append(" ")
+                        .append(getResources().getString(R.string.message_1))
+                        .append(" ")
+                        .append(getResources().getString(R.string.app_name))
+                        .append(" ")
+                        .append(getResources().getString(R.string.message_2))
+                        .append("\n\n");
```

Comment:
Why can't you replace this entire thing by `String.format` ? Seems like you can.